### PR TITLE
Resize vertex buffer if necessary in cf_mesh_update_vertex_data

### PR DESCRIPTION
**Pull request by waldnercharles**
I don't know if this is the right way to do this. It solved my problem so I figured I'd open a PR to discuss. Would it be better to instead do something like a pow2 resize?

### CODE REVIEW

**Comment by RandyGaul** [comment]
Looks good, though I would agree yes, a 2x size would be a bit better. Go ahead and just double the size if this if-statement is hit. The default size is very small like 5mb, so I would expect for games with lots of sprites for this to get hit a few times before settling.

File: src/cute_graphics.cpp [hunk 1]
```diff
@@ -1312,12 +1312,32 @@ void cf_mesh_set_attributes(CF_Mesh mesh_handle, const CF_VertexAttribute* attri
...
-	// @TODO Destroy/recreate for a larger size.
-
 	// Copy vertices over to the driver.
 	CF_MeshInternal* mesh = (CF_MeshInternal*)mesh_handle.id;
 	CF_ASSERT(mesh->attribute_count);
 	int size = count * mesh->vertices.stride;
+
+	// Resize buffers if necessary
+	if (size > mesh->vertices.size)
+	{
```
Comment (RandyGaul) [inline src/cute_graphics.cpp:1322]: Nit: Brace on same line as predicate

File: src/cute_graphics.cpp [hunk 2]
```diff
@@ -1312,12 +1312,32 @@ void cf_mesh_set_attributes(CF_Mesh mesh_handle, const CF_VertexAttribute* attri
 
 void cf_mesh_update_vertex_data(CF_Mesh mesh_handle, void* data, int count)
 {
-	// @TODO Destroy/recreate for a larger size.
-
 	// Copy vertices over to the driver.
 	CF_MeshInternal* mesh = (CF_MeshInternal*)mesh_handle.id;
 	CF_ASSERT(mesh->attribute_count);
 	int size = count * mesh->vertices.stride;
```
Comment (RandyGaul) [inline src/cute_graphics.cpp:1318]: Let's just add `* 2` to the end of this here

File: src/cute_graphics.cpp (hunk 1 above)
Comment (pusewicz) [reply src/cute_graphics.cpp]: Speaking of this, @RandyGaul how do you feel about adding linters and formatters so the code is always consistent?

Comment (RandyGaul) [reply src/cute_graphics.cpp]: Ehh not the biggest fan since it takes a lot of time to setup and maintain

Comment (pusewicz) [reply src/cute_graphics.cpp]: But then it makes life easier, and with github actions it can be validates with each PR.

**Pull request merged**